### DISCUSSION TITLE
Finding required libraries for this project

### DISCUSSION THREAD

**Discussion by souravlal** [marcel-licence/esp32_midi_sampler#20] [Q&A]
Hey, You have mentioned a few libraries , but I could not find any one of those libraries in arduino library manager.
Please help me.

**Comment by marcel-licence** [comment]
![grafik](https://user-images.githubusercontent.com/65245426/126048143-42c7b51b-2358-4905-83ab-7b32e5d59a38.png)
![grafik](https://user-images.githubusercontent.com/65245426/126048155-09010394-6348-47b0-95c4-2daaee16df05.png)
AC101 via GitHub -> https://github.com/marcel-licence/AC101
![grafik](https://user-images.githubusercontent.com/65245426/126048190-fbb620a1-5fe5-4846-8c51-a8d77fa18e5e.png)
![grafik](https://user-images.githubusercontent.com/65245426/126048195-f940c219-adf4-49c8-893d-8fe969869f7a.png)

**Reply by souravlal** [reply]
Thanks I have installed  all of them.
